runtime-transaction/src/runtime_transaction.rs

```diff
@@ -66,7 +54,7 @@ impl<T: StaticMetaAccess> StaticMeta for RuntimeTransaction<T> {
     }
 }
 
-impl<T: DynamicMetaAccess> DynamicMeta for RuntimeTransaction<T> {}
+impl<T: SVMMessage> DynamicMeta for RuntimeTransaction<T> {}
```

Comment:
dynamic meta available for any type of SVMMessage, this that's where we know to have resolved ATLs